### ISSUE TITLE
Zekvir abilities in standard Delves

### ISSUE THREAD

**Issue by patf0rd** [BigWigsMods/LittleWigs#1175]
Right now you only have Zekvir's abilities for his special Cave. Can you make the basic ones available for all Delves to have them up for Zekvir's surprise visits?

**Comment by ntowle** [comment]
Zekvir's abilities are already tracked in the regular Delves. The options for them are in the Rares module which shows in the dropdown for every Delve.

Are you saying the alerts are not working? Or were you just missing the options for them?

**Comment by patf0rd** [comment]
Ah it was moved here, sorry for putting it on BigWigs.

Thanks for pointing to these options, yes I missed them. I set up specific sounds for Zekvir's Cave (e.g. for "Enfeebling Spittle") and when I didn't hear them in a standard delve I thought it was not supported. But I also have to add that once in the Rares module (thankfully it's shared for standard delves).

So the only QoL change would be that the Rares module for Zekvir grabs the settings from the Zekvir Cave (and maybe a note there that it happens) but otherwise, it's fine for me, thank you.
